controllers/integration/integration_adapter.go

```diff
@@ -744,6 +732,32 @@ func (a *Adapter) createMissingReleasesForReleaseLinks(releaseLinks *[]releasev1
 	return nil
 }
 
+// getRequiredIntegrationTestScenariosForApplication returns the IntegrationTestScenarios used by the application being processed.
+// A IntegrationTestScenarios will only be returned if it has the
+// release.appstudio.openshift.io/optional label set to true or if it is missing the label entirely.
+func (a *Adapter) getRequiredIntegrationTestScenariosForApplication(application *hasv1alpha1.Application) (*[]v1alpha1.IntegrationTestScenario, error) {
+	labelSelector := labels.NewSelector()
+	integrationList := &v1alpha1.IntegrationTestScenarioList{}
+	labelRequirement, err := labels.NewRequirement("release.appstudio.openshift.io/optional", selection.NotIn, []string{"false"})
```

Comment:
The prefix four integration service labels is `test` so it would need to be:
`release.appstudio.openshift.io/optional` -> `test.appstudio.openshift.io/optional`

Reply:
Sorry, forgot about that.